include/mbgl/ios/MGLMapView.h

```diff
@@ -82,6 +82,14 @@ IB_DESIGNABLE
 /** The button shown in the lower-right of the map which when pressed displays the map attribution information. */
 @property (nonatomic, readonly) UIButton *attributionButton;
 
+/** Query the visible map for features at a given point. 
+*
+*   @warning Beta API. Return value subject to change in subsequent releases.
```

Comment:
If we’re considering this a “beta API”, I think it would be better to leave it undocumented (actually, just convert this block to a normal comment).